net/natmap/Makefile

```diff
@@ -0,0 +1,38 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=natmap
+PKG_VERSION:=20221111
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/heiher/natmap.git
+PKG_SOURCE_DATE:=2022-11-11
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_MIRROR_HASH:=4359bdddd16d7f97af4ee03f814ea4e649554c30eeed93e5f4fc586151d2476e
+
+PKG_MAINTAINER:=Richard Yu <yurichard3839@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=License
+
+PKG_USE_MIPS16:=0
```

Comment:
if this program can be compiled parallelly, please consider adding `PKG_BUILD_PARALLEL:=1`

https://github.com/heiher/natmap/blob/0ff5125dd04386d0abed2d8e0d2fa6875e21fbd4/build.mk#L10
given that `.git` will not be packed into tarball anyway, you may wanna manually pass `COMMIT_ID` cflag.